docs/rest-api.yaml

```diff
@@ -0,0 +1,106 @@
+swagger: "2.0"
```

Comment:
Do we need both raml and yaml?

Reply:
No, we switched from RAML to [OpenAPI](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).

Reply:
https://github.com/driebit/ginger/pull/394/commits/0daf23fe77324866321b09605fd7b6fcec4f420c